daemon/api.go

```diff
@@ -634,21 +638,29 @@ func (inst *snapInstruction) Agreed(intro, license string) bool {
 var snappyInstall = snappy.Install
 
 func (inst *snapInstruction) install() interface{} {
+	// FIXME: handle license agreement
```

Comment:
The TODO here should be above the commented out code at the end. Otherwise it looks like code left out by mistake.